.github/workflows/deploy-prod.yml

```diff
@@ -10,7 +10,7 @@ env:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 20
```

Comment:
This issue originated from TF didn't it? Increasing to 20 minutes didn't help us and will probably just waste more time in the future, if deploys are taking more than 10 minutes they're probably hanging right?

Reply:
CF update is taking for time 5-6m. Previously we executed CF update from local machine ....